### DISCUSSION TITLE
Compatibility with CARLA 0.9.12

### DISCUSSION THREAD

**Discussion by rsuwa** [praveen-palanisamy/macad-gym#53] [Q&A] [answered]
Does it work with CARLA 0.9.12? If not, how much work do you expect to have to do to fix it?

**Comment by praveen-palanisamy** [comment, marked as answer]
Hi @rsuwa ,
I was told by @lcipolina that [MACAD-Gym](https://github.com/praveen-palanisamy/macad-gym) works with CARLA 0.9.12. I am yet to test it specially with 0.9.12 but if you can try it out and let me know if you run into any issues, it will be appreciated.

**Comment by rsuwa** [comment]
@praveen-palanisamy Thank you for sharing the information! I'll let you know if any issues happen!
